Qt5Test is only needed for running the regression tests.

--- CMakeLists.txt.orig	2018-04-25 11:23:10 UTC
+++ CMakeLists.txt
@@ -258,7 +258,9 @@ if (NOT WITH_QT4 AND NOT Qt4_FOUND AND NOT QT4_FOUND)
       set(Qt5Xml_DIR "${_qt5Dir}/lib/cmake/Qt5Xml")
       set(Qt5Multimedia_DIR "${_qt5Dir}/lib/cmake/Qt5Multimedia")
       set(Qt5LinguistTools_DIR "${_qt5Dir}/lib/cmake/Qt5LinguistTools")
-      set(Qt5Test_DIR "${_qt5Dir}/lib/cmake/Qt5Test")
+      if (BUILD_TEST_APP)
+        set(Qt5Test_DIR "${_qt5Dir}/lib/cmake/Qt5Test")
+      endif (BUILD_TEST_APP)
       if (WITH_DBUS)
         set(Qt5DBus_DIR "${_qt5Dir}/lib/cmake/Qt5DBus")
       endif (WITH_DBUS)
@@ -299,9 +301,11 @@ if (NOT WITH_QT4 AND NOT Qt4_FOUND AND NOT QT4_FOUND)
       endif (Qt5${_qt5Component}_FOUND)
     endforeach ()
     find_package(Qt5LinguistTools)
-    find_package(Qt5Test)
-    set(QT_QTTEST_LIBRARY Qt5::Test)
-    set(QT_QTTEST_INCLUDE_DIR ${Qt5Test_INCLUDE_DIRS})
+    if (BUILD_TEST_APP)
+      find_package(Qt5Test)
+      set(QT_QTTEST_LIBRARY Qt5::Test)
+      set(QT_QTTEST_INCLUDE_DIR ${Qt5Test_INCLUDE_DIRS})
+    endif (BUILD_TEST_APP)
     if (Qt5DBus_FOUND)
       set(HAVE_QTDBUS 1)
     endif (Qt5DBus_FOUND)
